integration_tests/e2e/goal/addAndEditNote.cy.ts

```diff
@@ -0,0 +1,60 @@
+import Page from '../../pages/page'
+import AddStepPage from '../../pages/goal/AddStepPage'
+import AddNotePage from '../../pages/goal/AddNotePage'
+import ReviewPage from '../../pages/goal/ReviewPage'
+import OverviewPage from '../../pages/overview/OverviewPage'
+
+context('Add and edit a step', () => {
```

Comment:
Minor, copy and paste typo 😁  (`Add and edit a step` should be `Add and edit a note` ?)

Reply:
Me being lazy - fixed!